src/SmartComponents/Remediations/steps/ExistingOrNew.js

```diff
@@ -0,0 +1,107 @@
+import React, { Component } from 'react';
+import { getRemediations } from '../../../api/remediations';
+
+import {
+    Form,
+    FormGroup,
+    TextInput
+} from '@patternfly/react-core';
+
+class NameStep extends Component {
+
+    constructor (props) {
+        super(props);
+        this.state = {
+            isNew: true,
+            name: '',
+            existing: false,
+            selectedRemediation: false
+        };
+    };
+
+    onTextChange = (name) => {
+        this.setState({ name });
+    };
+
+    onRadioChange = (isNew) => {
+        this.setState({
+            isNew,
+            selectedRemediation: isNew ? false : this.state.existing[0]
+        });
+    }
+
+    onRemediationSelected = (event) => {
+        this.setState({
+            selectedRemediation: this.state.existing.find(existing => existing.id === event.target.value)
+        });
+    };
+
+    componentDidMount () {
+        getRemediations()
+        .then(({ remediations }) => this.setState({ existing: remediations }));
+    }
+
+    render() {
+
+        const { name, isNew, existing } = this.state;
+
+        return (
+            <React.Fragment>
+                <h1>Do you want to modify an existing Remediation or create a new one?</h1>
+                <Form>
+                    <div>
+                        <label>
+                            <input
+                                type="radio"
```

Comment:
Let's use the Patternfly inputs:
https://patternfly-react.surge.sh/patternfly-4/components/radio